container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2149,9 +2153,6 @@ instance_groups:
           memory: 256
           virtual-cpus: 1
           service-account: eirini
-  configuration:
-    templates:
-      properties.eirini.run_cert_copier: ""
```

Comment:
Why are you removing this? This setting prevents eirini from deploying the cert_copier. If you remove this setting, then you have to update eirini-bosh-release to unconditionally stop deploying the cert_copier (for SCF).

Reply:
Oh, I see you have already done this in https://github.com/cloudfoundry-community/eirini-bosh-release/commit/f2856a00bdc234e3437bff61888b5d5df64e4dc4